src/pages/Detail/Reservation.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Reservation(props) {
```

Comment:
```suggestion
function Reservation({openModal}) {
```
매개변수자리에서 구조분해할당 가능합니다

Reply:
반영했습니다~!